src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String typeIcon;
+    protected String deadline = "";
+
+    public Task(String description) {
+        if(description.contains("todo")||description.contains("deadline")||description.contains("event")) {
+            String command = description.substring(0, description.indexOf(' '));
+            String content = description.substring(description.indexOf(' ') + 1);
+
+            this.description = content;
+            this.isDone = false;
+            switch (command) {
+            case "todo":
+                typeIcon = "T";
+                break;
+            case "deadline":
+                typeIcon = "D";
+                deadline = content.substring(content.indexOf('/')+4);
+                break;
+            case "event":
+                typeIcon = "E";
+                deadline = content.substring(content.indexOf('/')+4);
+                break;
+            default:
+                typeIcon = " ";
+            }
+        }else{
+            this.description = description;
+            this.isDone = false;
+        }
+    }
+
+    public String getStatusIcon() {
+        //return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
I think these comments could be deleted if they are no longer needed? Personally I like to keep these code fragments too in case I need them later, but in a separate draft document .